server.js

```diff
@@ -29,9 +29,15 @@ app.use(setNewToken);
 
 // 404 page
 app.use(function (req, res) {
-  res.render("404", { title: "Oops! Page not found." });
+  res.render("404", {
+    title: "Oops! Page not found.",
+    partials: "404",
```

Comment:
What's with this?

Reply:
that's called - Lara was lost, but then reworked it all